### ISSUE TITLE
Php2Xml dynamic property exception

### ISSUE THREAD

**Issue by GameCharmer** [intuit/QuickBooks-V3-PHP-SDK#486]
```
Creation of dynamic property QuickBooksOnline\API\XSD2PHP\src\com\mikebevz\xsd2php\Php2Xml::$rootNsName is deprecated
```

Path: quickbooks/v3-php-sdk/src/XSD2PHP/src/com/mikebevz/xsd2php/Php2Xml.php(181)

**Comment by sujitharamadass** [comment]
@GameCharmer The fix will be released soon.

**Comment by sujitharamadass** [comment]
Fixed in version 6.1.2
